PlaywrightUltimate/PageObjects/UltimateQaPage.cs

```diff
@@ -0,0 +1,88 @@
+﻿using Microsoft.Playwright;
+
+namespace PlaywrightUltimate.PageObjects
+{
+    internal class UltimateQaPage(IPage page)
+    {
+        private const string UltimateQaTargetPage = "https://courses.ultimateqa.com/enrollments";
+
+        // Locators
+
+        private ILocator EmailInput => page.Locator("input[name='user[email]']");
+        private ILocator PasswordInput => page.Locator("input[name='user[password]']");
+        private ILocator SignInButton => page.Locator("button[type='submit']");
+        
+        private ILocator MyDashboard => page.GetByRole(AriaRole.Link, new PageGetByRoleOptions { Name = "My Dashboard" });
+        private ILocator ViewMoreCoursesLink => page.GetByRole(AriaRole.Link, new PageGetByRoleOptions { Name = "View more courses" });
+        private ILocator ProductsSearch => page.GetByPlaceholder("Search");
+        
+        private ILocator UserMenu => page.GetByLabel("Toggle menu");
+        private ILocator DashboardHeader(string userName) => page.GetByRole(AriaRole.Heading, new PageGetByRoleOptions { Name = $"      Welcome back, {userName}!    " });
+        
+        // Methods
+
+        public async Task GoToUltimateQaPage()
```

Comment:
this method is never used. It is more correct to use this method than to use URL hardcoded in the test, but you need to remove Assert from this method as opening this page is not always result in /sign_in

Reply:
this method is breadcrumb from initial implementation according to the homework task, I will remove it